extensions/azure-functions-resteasy/runtime/src/main/java/io/quarkus/azure/functions/resteasy/runtime/Function.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.azure.functions.resteasy.runtime;
+
+import java.util.Optional;
+
+import javax.ws.rs.ext.RuntimeDelegate;
+
+import org.jboss.resteasy.core.SynchronousDispatcher;
+import org.jboss.resteasy.mock.MockHttpResponse;
+import org.jboss.resteasy.specimpl.ResteasyUriInfo;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+import com.microsoft.azure.functions.HttpStatus;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
+
+import io.quarkus.runtime.Application;
+
+public class Function {
+
+    public HttpResponseMessage run(
+            @HttpTrigger(name = "req") HttpRequestMessage<Optional<byte[]>> request,
+            final ExecutionContext context) {
+        if (request.getUri().getPath().endsWith("quarkus-status")) {
```

Comment:
What is the purpose of this? It does not seem to be documented anywhere or used in the rest of the code?

Reply:
There for debugging bootstrap.